package.json

```diff
@@ -55,17 +55,22 @@
 		"autogypi.json"
 	],
 	"devDependencies": {
+		"@types/node": "^8.10.17",
 		"ava": "^0.25.0",
 		"clang-format": "^1.2.2",
 		"humanize": "0.0.9",
 		"husky": "^0.14.3",
-		"proc-stats": "0.0.4"
+		"prettier": "^1.12.1",
```

Comment:
Why prettier? We already use clangformat to check style, I think prettier would clash with that: it probably format the code in a way that clangformat does not accept

Reply:
I use prettier because it will automatically add ';', clang-format does not

Reply:
before submit, I will run lint to make sure code will be formatted with clang-format

Reply:
> I use prettier because it will automatically add ';', clang-format does not

Yes, and I also miss some other checks that was provided by eslint, but clangformat has the advantage that also format objective-c c and c++. We'll need to improve on this. Meanwhile, you can jsut install it globally?

Reply:
OK, I see.